deploy/scripts/src/btm/test.ts

```diff
@@ -102,21 +204,62 @@ const awsConfig = new AWSConfig({
 })
 
 const eventData = {
-  payload: __ENV.DATA_BTM_SQS_PAYLOAD
+  payloadEventsString: __ENV.DATA_BTM_SQS_PAYLOAD_EVENTS,
+  payloadTimestamp: __ENV.DATA_BTM_SQS_PAYLOAD_TIMESTAMP
 }
 
+const payloadEventsArray = JSON.parse(eventData.payloadEventsString)
+
+const payloadTimestampArray = eventData.payloadTimestamp.split(',')
+const payloadTimestampMin: number = Number(payloadTimestampArray[0])
+const payloadTimestampMax: number = Number(payloadTimestampArray[1])
+
+let messageBody: string = ''
+let randomTimestamp: number
+let timestampFormatted: string
```

Comment:
These need to be moved to be declared in their respective functions as I think the scope will cause issues with the multiple VUs

Reply:
Done

Reply:
This still needs to be changed